integration/mediation-tests/tests-mediator-1/src/test/java/org/wso2/carbon/esb/mediator/test/property/PropertyIntegrationOperationScopeRemovePropertiesTestCase.java

```diff
@@ -0,0 +1,149 @@
+                "Proxy Invocation Failed!");
+        assertTrue(isMatchFound("symbol = WSO2 Lanka"),
+                "String Property Not Either Set or Removed in the Axis2 scope!!");
+    }
+
+    @Test(groups = "wso2.esb",
+          description = "Remove action as \"value\" and type Float (operation scope)")
+    public void testFloatVal() throws Exception {
+        logViewer.clearLogs();
+        OMElement response = axis2Client
+                .sendSimpleStockQuoteRequest(getProxyServiceURLHttp("propertyFloatOperationRemoveTestProxy"), null, "Random Symbol");
+        assertTrue(response.toString().contains("Property Set and Removed"),
+                "Proxy Invocation Failed!");
+        assertTrue(isMatchFound("symbol = 123.123"),
+                "Float Property Not Either Set or Removed in the Axis2 scope!!");
+    }
+
+    @Test(groups = "wso2.esb",
+          description = "Remove action as \"value\" and type Long (operation scope)")
+    public void testLongVal() throws Exception {
+        logViewer.clearLogs();
+        OMElement response = axis2Client
+                .sendSimpleStockQuoteRequest(getProxyServiceURLHttp("propertyLongOperationRemoveTestProxy"), null, "Random Symbol");
+        assertTrue(response.toString().contains("Property Set and Removed"),
+                "Proxy Invocation Failed!");
+        assertTrue(isMatchFound("symbol = 123123123"),
+                "Long Property Not Either Set or Removed in the Axis2 scope!!");
+    }
+
+    @Test(groups = "wso2.esb",
+          description = "Remove action as \"value\" and type Short (operation scope)")
+    public void testShortVal() throws Exception {
+        logViewer.clearLogs();
+        OMElement response = axis2Client
+                .sendSimpleStockQuoteRequest(getProxyServiceURLHttp("propertyShortOperationRemoveTestProxy"), null, "Random Symbol");
+        assertTrue(response.toString().contains("Property Set and Removed"),
+                "Proxy Invocation Failed!");
+        assertTrue(isMatchFound("symbol = 12"),
+                "Short Property Not Either Set or Removed in the Axis2 scope!!");
+    }
+
+    @Test(groups = "wso2.esb",
+          description = "Remove action as \"value\" and type OM (operation scope)")
+    public void testOMVal() throws Exception {
+        logViewer.clearLogs();
+        OMElement response = axis2Client
+                .sendSimpleStockQuoteRequest(getProxyServiceURLHttp("propertyOMOperationRemoveTestProxy"), null, "Random Symbol");
+        assertTrue(response.toString().contains("Property Set and Removed"),
+                "Proxy Invocation Failed!");
+        assertTrue(isMatchFound("symbol = OMMMMM"),
+                "OM Property Not Either Set or Removed in the Axis2 scope!!");
+    }
+
+    /**
+     * The method that checks whether the particular
+     * match string is available in the sysytem logs
+     */
+    private boolean isMatchFound(String matchStr) throws Exception {
```

Comment:
Arn't we using the same match used in PropertyIntegrationAxis2ScopeRemovePropertiesTestCase.java ? shall we remove the duplicated lines ?

Reply:
This is a private method. Also we can't move this to a common util class because these checks are specific only to property mediator tests.